core/game_object.lua

```diff
@@ -2148,9 +2146,85 @@ Set `prefix_config.key = false` on your object instead.]]):format(obj.key), obj.
             SMODS.process_loc_text(G.localization.descriptions.Other, self.key, self.loc_txt, 'description')
             SMODS.process_loc_text(G.localization.misc.labels, self.key, self.loc_txt, 'label')
         end,
-        inject = function() end,
+        inject = function(self)
+            G.shared_stickers[self.key] = Sprite(0, 0, G.CARD_W, G.CARD_H, G.ASSET_ATLAS[self.atlas], self.pos)
+        end,
+        -- relocating sticker checks to here, so if the sticker has different checks than default
+        -- they can be handled without hooking/injecting into create_card
+        -- or handling it in set_sticker
+        -- TODO: rename
+        sticker_check = function(self, card, center, area)
+            if (
+                center[self.key..'_compat'] or -- explicit marker
+                (self.default_compat and not self.compat_exceptions[center.key]) or -- default yes with no exception
+                (not self.default_compat and self.compat_exceptions[center.key]) -- default no with exception
+            ) and 
+            (not self.needs_enable_flag or G.GAME.modifiers['enable_'..self.key]) and
+            pseudorandom((area == G.pack_cards and 'packssj' or 'shopssj')..self.key..G.GAME.round_resets.ante) > (1-self.rate)
+            then
+                return true
+            end
+        end,
+        set_sticker = function(self, card, val)
+            card.ability[self.key] = val
+        end
+    }
+
+    -- Create base game 
+    -- sticker_check is set to nil on all since they'd use base game checks for application
+    SMODS.Sticker{
+        key = "eternal",
+        badge_colour = HEX 'c75985',
+        prefix_config = {key = false},
+        pos = { x = 0, y = 0 },
+        sticker_check = nil, 
```

Comment:
`sticker_check = nil` does nothing, it acts like there is no value there. Use 0 or false instead. Though the base checks should work here?

Reply:
Thunk already has sticker checks for these 3 and does some funkyness with the sticker check for eternal/perishable (they use the same pseudorandom var), so sticker_check function is removed as to preserve base game sticker application. I'll just remake the sticker check function instead (assuming eternal/perishable funkyness doesn't stop that).

Reply:
Update: psuedorandom does change when calling the function again therefore adding `sticker_check` as a function would mean different RNG for their application. Not sure on the opinion for this, so defaulting to `sticker_check = false` for both.